integer_birth_year.rb

```diff
@@ -1,9 +1,12 @@
-# Ask for a person’s age, and then calculate and display what year they were born in.
+# Ask for a person’s age.
+# Using the Date class and the current year, calculate and display what year they were born in.
+# 
+# Ex:
 # 
 # Input:
 #   80
 # 
 # Key output:
-#   "Wow, you were born in 1940. You're old!"
+#   "Wow, you were born in 1943. You're old!"
```

Comment:
To be extra clear, the comment could also say something like:

```rb
# Key output (assuming the current year is 2023):
#   "Wow, you were born in 1943. You're old!"
```


I think we talked about moving all of the instructions to the `README` so it's easier to keep track of updates. It also gets students into the habit of reading the `README` which has been a concern.

Reply:
When do we want to start moving these down into the `README`'s ?

Reply:
I don't think there was ever a hard decision on moving instructions to `README`s. It was just an idea that was proposed at one point.~~At the very least, I don't think it's urgent seeing as students have already done this assignment this quarter.~~ I think the next session of DPI starts soon actually. @raghubetina did you have an opinion on whether the instructions should be moved to the README? [This was suggested during DPI](https://www.notion.so/each_letter_count-rb-suggestion-c76cc05eb94a434b973a737730c5eb68) since it's hard to put good examples in Ruby comments.

Reply:
Ah, good catch. Let's think more about moving notes to the README, but merge this change for now.